src/plugins/autoColumnSize/test/autoColumnSize.e2e.js

```diff
@@ -482,4 +482,30 @@ describe('AutoColumnSize', () => {
 
     expect(spy.calls.mostRecent().args[5]).toEqual([{ id: 1000 }]);
   });
+
+  it('should not change width and widths after select/click cell', () => {
```

Comment:
I think that this test doesn't cover this bug correctly. When I attached the older version of the handsontable to this test it passes, should fail. Can you check that?